apps/coscrad-frontend/src/components/resources/terms/term-index-table.tsx

```diff
@@ -135,7 +99,7 @@ export const TermIndexTable = ({
      * and pagination logic.
      */
     const table =
-        paginatedData.length === 0 ? (
+        tableData.length === 0 ? (
             <NotFoundPresenter />
         ) : (
             <Box sx={{ width: '100%' }}>
```

Comment:
Terms heading repeats here and somewheres else

Reply:
Great. Thanks for the reminder. I'll write a Cypress test for this regression.

Reply:
Turned out to be a hard thing to test with Cypress. But I have visually verified via the Cypress run that this has been fixed.